### PR TITLE
Correct OOZIE_SERVER_PATH

### PR DESCRIPTION
**Pull request by kiiranh**
This fixes issue #216.

### CODE REVIEW

File: cookbooks/bcpc-hadoop/recipes/oozie.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 OOZIE_LIB_PATH='/usr/hdp/current/oozie'
 OOZIE_CLIENT_PATH='/usr/hdp/current/oozie-client'
-OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie-server'
+OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie/oozie-server'
```
Comment (amithkanand) [inline cookbooks/bcpc-hadoop/recipes/oozie.rb]: Can we use `/usr/hdp/current/oozie-server/oozie-server` instead of hard coding the `HDP` version?

Comment (kiiranh) [inline cookbooks/bcpc-hadoop/recipes/oozie.rb]: We can indeed, just that the HDP version is hard coded in various components throughout the code base. I will make the change here though

**Comment by kiiranh** [comment]
Code  updated and tested. Please review/merge

**Comment by amithkanand** [comment]
LGTM

**Pull request merged**